craftcms/composer.json

```diff
@@ -17,7 +17,7 @@
     "craftcms/redactor": "2.8.8",
     "google/cloud-storage": "^1.24",
     "rosas/nested-entries-graphql-queries": "^1.0",
-    "rosas/universal-dam-integrator": "dev-main",
+    "rosas/universal-dam-integrator": "dev-EPO-6120",
```

Comment:
I don't _think_ this will matter, but `dev-main` should be fine, I just deleted the `EPO-6120` branch after the merge

Reply:
shoot.  Unintended.  Good catch.  Fixing it.